.idea/runConfigurations/Artemis__Server__Athene_.xml

```diff
@@ -5,9 +5,9 @@
     <option name="ACTIVE_PROFILES" value="dev,bamboo,bitbucket,jira,artemis,scheduling,athene,local" />
     <option name="VM_PARAMETERS" value="--illegal-access=warn" />
     <option name="ALTERNATIVE_JRE_PATH" />
-    <option name="SHORTEN_COMMAND_LINE" value="NONE" />
+    <option name="SHORTEN_COMMAND_LINE" value="MANIFEST" />
```

Comment:
Are you sure you want to push this change? (I am not familiar with Athene, but if it is a similar issue like the one where the Artemis server wouldn't start until you add a shortened command-line option e.g. in IntelliJ. 
I think this is a custom/local-machine issue that should not be pushed?) (aka. maybe you commited&pushed this file by accident?)

Reply:
Yep, this is definitely an unintentional change, only local to my machine. accidental. Thanks for noticing 👍 .